tmt/cli.py

```diff
@@ -411,6 +411,10 @@ def create(context, name, template, force, **kwargs):
 @click.option(
     '--makefile / --no-makefile', default=True,
     help='Convert Beaker Makefile metadata')
+@click.option(
+    '--type', metavar='TYPE', default='multihost',
+    help='Convert Type from Beaker Makefile to the tag section. The default '
+         'value is "multihost". Use "all" to convert all detected types.')
```

Comment:
I think it would be useful to support providing multiple types as well, e.g.

    tmt test import --type multihost --type sanity

What do you think?